gemrb/core/GUI/GameControl.cpp

```diff
@@ -608,7 +611,9 @@ void GameControl::DrawSelf(Region screen, const Region& /*clip*/)
 	Point gameMousePos = GameMousePos();
 	// draw reticles
 	if (isFormationRotation) {
-		double angle = AngleFromPoints(gameMousePos, gameClickPoint);
+		double angle = formationBaseAngle;
+		if (Distance (gameMousePos, gameClickPoint) > 5)
+			angle = AngleFromPoints(gameMousePos, gameClickPoint);
```

Comment:
No spaces before opening parens and braces to avoid two types of warnings (ambiguity due to following statement)
```suggestion
		if (Distance(gameMousePos, gameClickPoint) > 5) {
			angle = AngleFromPoints(gameMousePos, gameClickPoint);
		}
```

Reply:
I don't see any ambiguity, but ok.